### PR TITLE
Arachne 2422 Cannot reset optional metadata setting to nothing

### PR DESCRIPTION
**Pull request by aklochkova**

### CODE REVIEW

File: src/components/FiltersList/presenter.jsx [hunk 1]
```diff
@@ -54,32 +58,33 @@ function FilterFormSelect(props) {
...
-      if (isMulti) {
-        const isAnyOptionSelected = val.includes(anyOptionValue);
-        newValue = isAnyOptionSelected ? [] : val;
-      } else {
-        // whether 'Any' option selected
-        newValue = val === anyOptionValue ? null : val;
-      }
-      input.onChange(newValue);
-    }}
+      isMulti={isMulti}
```
Comment (pavgra) [inline src/components/FiltersList/presenter.jsx]: Remove all dummy-formatting related diff

File: src/modules/DataCatalog/components/ViewEdit/Edit/AttributesList/container.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ import SelectorsBuilder from './selectors';
 
 const selectors = (new SelectorsBuilder()).build();
 
-/** @augments { Component<any, any> } */
+/** @arguments { Component<any, any> } */
```
Comment (johnSamilin) [inline src/modules/DataCatalog/components/ViewEdit/Edit/AttributesList/container.js]: augments, this is typescript directive

File: src/components/FiltersList/presenter.jsx (hunk 1 above)
Comment (aklochkova) [reply src/components/FiltersList/presenter.jsx]: I've applied .eslintrc rules stored in project but there are some diffs still

File: src/modules/DataCatalog/components/ViewEdit/Edit/AttributesList/presenter.jsx [hunk 3]
```diff
@@ -42,29 +47,29 @@ function ImmutableAttribute({ name, value }) {
...
 export function AttributesFormListItem({
-  item,
-  input,
-  meta,
-  isWide = false,
-  disabled,
-  useAutocomplete,
-  autocompleteOptions,
-}) {
+                                         item,
```
Comment (johnSamilin) [inline src/modules/DataCatalog/components/ViewEdit/Edit/AttributesList/presenter.jsx]: fix indentation

File: src/services/Utils.js [hunk 4]
```diff
@@ -243,19 +269,20 @@ class Utils {
...
-    Component,
-    mapStateToProps = null,
-    getMapDispatchToProps = null, // if it should be defined as Object (this is done due to unavailability of extending of getters in TS)
-    mapDispatchToProps = null, // if it should be defined as Function
-    mergeProps = null,
-    getModalParams = null,
-    getFormParams = null,
-    getFetchers = null,
-  } = {}) {
+                                   Component,
```
Comment (johnSamilin) [inline src/services/Utils.js]: fix indentation

**Pull request merged**